doc/overview.md

```diff
@@ -6,17 +6,24 @@
 
 A Learning Record Store (LRS) is a persistent store for xAPI statements and associated attachments and documents. The full LRS specification can be found in Part 3 of the [xAPI specification](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md). SQL LRS is distinct from other LRSs developed at Yet Analytics for being SQL-based and supporting multiple SQL database management systems (DBMSs) like SQLite, and Postgres.
 
-### How to use SQL LRS?
+In addition, SQL LRS includes several additional features, such as:
+- A UI that features API key and admin management, statement browsing, and much more
+- A [Docker image](docker.md)
+- Custom [authority configuration](authority.md)
+- Authentication via [OpenID](oidc.md)
+- Support for BI platforms like [Apache Superset](superset.md)
+- [Reactions](reactions.md)
 
-SQL LRS Accounts can be created using the user interface (see [Getting Started](startup.md)) or API calls (see [Endpoints](endpoints.md)). Accounts can be used to create or access SQL LRS credentials. These credentials, which consist of an API key, a secret API key, and their scopes ([described in the xAPI spec](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#42-oauth-10-authorization-scope)), are then used as headers for LRS-specific methods to authenticate and authorize the request sender.
+### What is SQL LRS not?
 
-### Differences from Cloud LRS
+SQL LRS, as an LRS, is purely a database storage app for xAPI statements, documents, and other related data. It is _not_ a Learning Management System (LMS) that generates xAPI statements from learning data, nor is it a combined platform that includes an LMS.
 
-If you previously used Yet's Cloud LRS products, it is important to be aware of certain differences:
+In addition, there are several features that SQL LRS does not support (but were supported by Yet's past Cloud LRS products), such as:
```

Comment:
Not sure if it was appropriate to point out that our cloud LRS products are now deprecated, so I just said "past"